spec/fixtures/markdown_formatter/expected.md

```diff
@@ -0,0 +1,139 @@
+# RuboCop Inspection Report
+
+3 files inspected, 22 offenses detected:
+
+### app/controllers/application_controller.rb - 1 offense
+  **Line # 1 - convention:** Style/FrozenStringLiteralComment: Missing frozen string literal comment.
```

Comment:
I'm thinking each offense should probably be a bullet-point with the code snippet nested underneath.